playbook/app/pb_kits/playbook/pb_filter/_filter.scss

```diff
@@ -67,15 +67,13 @@
 }
 .maskContainer::after {
     content: " ";
-    background-image: linear-gradient(to right, rgba($card_light,.3) , rgba($card_light,1));
```

Comment:
@kre8sions What do these gradients do?  They're showing up in the dark mode version.  Do they need to be conditionally rendered for only light mode?

Reply:
@bh247484 they are need to fade the side scroll

Reply:
Im not sure. When I deleted them, nothing happens on my end for Rails or React. The white gradient space goes away in dark mode but the formatiing seems to stay the same. Thoughts??